code-challenges.test.js

```diff
@@ -14,47 +13,150 @@
+
+
 // --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
 // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const studyMinutesWeek1 = {
-  sunday: 90,
-  monday: 30,
-  tuesday: 20,
-  wednesday: 15,
-  thursday: 30,
-  friday: 15,
-  saturday: 60
-}
-// Expected output: [15, 15, 20, 30, 30, 60, 90]
-
-const studyMinutesWeek2 = {
-  sunday: 100,
-  monday: 10,
-  tuesday: 45,
-  wednesday: 60,
-  thursday: 20,
-  friday: 15,
-  saturday: 65
-}
+
+  // describe("lowTogrt", () => {
+  //   it("returns an array of the values sorted from least to greatest.", () => {
+  //     expect(lowTogrt(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90]),
+  //     expect(lowTogrt(studyMinutesWeek2)).toEqual([10, 15, 20, 45, 60, 65, 100])
+  //   })
+  // })
+
+  // Good Fail:
+  // FAIL  ./code-challenges.test.js
+  // lowTogrt
+  //   ✕ returns an array of the values sorted from least to greatest. (1 ms)
+
+  // ● lowTogrt › returns an array of the values sorted from least to greatest.
+
+  //   ReferenceError: lowTogrt is not defined
+
+
+// const studyMinutesWeek1 = {
+//   sunday: 90,
+//   monday: 30,
+//   tuesday: 20,
+//   wednesday: 15,
+//   thursday: 30,
+//   friday: 15,
+//   saturday: 60
+// }
+// // Expected output: [15, 15, 20, 30, 30, 60, 90]
+
+// const studyMinutesWeek2 = {
+//   sunday: 100,
+//   monday: 10,
+//   tuesday: 45,
+//   wednesday: 60,
+//   thursday: 20,
+//   friday: 15,
+//   saturday: 65
+// }
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
 // b) Create the function that makes the test pass.
 
+  // Pseudo code:
+  // input: object
+  // output: returns an array of the values sorted from least to greatest.
+  // process:
+  // create a function name: lowTogrt
+  // use Object.values(object) to get all value from giving object
+  // then use .sort((a, b) => a - b) to arrange array value from least to greatest
+  // return Object.values(object).sort((a, b) => a - b)
+  // pass test
+
+      // const lowTogrt = (object) => {
+      //   return Object.values(object).sort((a, b) => a - b)
+      // }
+
+  
+  // pass test:
+  // PASS  ./code-challenges.test.js
+  //   lowTogrt
+  //     ✓ returns an array of the values sorted from least to greatest. (2 ms)
+
+
 // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+  describe("accSum", () => {
+    it("returns an array of the accumulating sum.", () => {
+      expect(accSum(accountTransactions1)).toEqual([100, -17, -23, -9]),
+      expect(accSum(accountTransactions2)).toEqual([250, -89, 100, -96]),
```

Comment:
Make sure you are paying attention to the expected output. This test currently is testing for the original input to be returned